common/source/docs/common-brushless-escs.rst

```diff
@@ -36,18 +47,97 @@ The DShot ESC protocol's key advantages are:
 - no need to do any ESC throttle range calibration
 - very high protocol frame rates are supported
 
-While DShot is available on many ESCs, those with BLHeli32 capability, offer the most performance and often, additional features such as ESC telemetry. In addition, ArduPilot provides special programming capability for BLHeli32 ESCs via its passthrough feature (See :ref:`common-dshot` section.)
+While DShot is available on many ESCs, those with BLHeli32 setup capability, offer the most performance and often, additional features such as ESC telemetry. In addition, ArduPilot provides special programming capability for BLHeli32 ESCs via its passthrough feature (See :ref:`common-blheli32-passthru` section.)
 
 .. note::
    Recently there is a growing number of proprietary and non-proprietary 16 / 32 bit ESCs with firmware that support DShot and other digital ESC protocols, but not BLHeli_32-specific features like passthrough and telemetry. See your ESC's manual for further detail on supported features.
 
-Where to buy
-============
+Configuration
+=============
+
+Protocol
+--------
+The ESC control protocol is selected by the :ref:`MOT_PWM_TYPE <MOT_PWM_TYPE>` parameter on Copter and Rover, or :ref:`Q_M_PWM_TYPE <Q_M_PWM_TYPE>` on QuadPlanes for copter function motors. Bi-directional DShot is selected by selecting DShot in :ref:`MOT_PWM_TYPE <MOT_PWM_TYPE>` and then selecting the required channels through :ref:`SERVO_BLH_BDMASK <SERVO_BLH_BDMASK>`.
```

Comment:
We need to be a little careful about this reference to MOT_PWM_TYPE because the enum values are different for Copter and Rover.

Reply:
since we dont explicitly call out the values or even the ENUM names, we are alright